### DISCUSSION TITLE
How to registration

### DISCUSSION THREAD

**Discussion by KOOLAB1** [simnibs/simnibs#263] [2. General]
To view customized electrodes, how to register to the program after converting from another coordinate system to the RAS coordinate system? I think there is an affine conversion, but is that the only way to do it? I understand that the eeg position folder is created when creating an msh file. I am curious about which csv file you are converting to here or what method you are trying to convert it to.

ex) not 4x4 matrix -> x,y,z
i want to transform 3D coordinate -> x,y,z

**Comment by jduemose** [comment]
@KOOLAB1 I converted your issue to a discussion thread.

If your initial coordinates are in MNI space then you can use the command line function [mni2subject_coords](https://simnibs.github.io/simnibs/build/html/documentation/command_line/mni2subject_coords.html#mni2subject-coords) for example. This uses one of the transformation in the toMNI folder (from a charm run).

As far as I remember, the files in eeg_positions folder are transformed from mni space to subject space using the nonlinear deformation estimated by charm.

Best,
Jesper
